### ISSUE TITLE
Rename labmode?

### ISSUE THREAD

**Issue by jlstevens** [pyviz/nei#1]
There was some discussion about renaming labmode. 

Personally, I am fine with this name but there were some other suggestions: Scriven and Satern. As this project is not a replacement for all of Jupyter, I don't think the latter should be considered.

There is also one other name I have just found to consider: Scrive (shortens Scriven by one letter and looks more like scribe which isn't an archaic word).

Unfortunately, any renaming will be a big task as elisp doesn't not support namespaces leading to lots of long names such as ``labmode--open-websocket`` ('private') and ``labmode-view-notebook``. There are a lot of these and I would have to rename them to ``scrive--open-websocket`` ('private') and ``scrive-view-notebook`` etc. If I am convinced to rename the project, I'll do it as a PR.

**Comment by jbednar** [comment]
I'm fine with scrive.

**Comment by jlstevens** [comment]
Nearly a year on, I think I've finally settled on a name: NEI (Notebook Emacs Integration)

It is inspired by EIN which is *the* official way to work with notebooks in emacs - if you have the luxury of ignoring the web browser entirely. The name is short and available on MELPA and I've reserved it on PyPI.

I can be made to stand for 'Notebook Editor Integration' if it ever supports an editor other than emacs.
